src/coverlet.core/coverlet.core.csproj

```diff
@@ -8,9 +8,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Mono.Cecil" Version="0.10.4" />
+    <PackageReference Include="Mono.Cecil" />
     <!-- Do not upgrade this version or we won't support old SDK -->
-    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+    <PackageReference Include="Newtonsoft.Json" />
```

Comment:
Pls move the above comment in targets file

Reply:
Why not ref below?
```xml
   <!--
              Do not change System.Reflection.Metadata version since we need to support VSTest DataCollectors. Goto https://www.nuget.org/packages/System.Reflection.Metadata to check versions.
              We need to load assembly version 1.4.2.0 to properly work
              We can check minimum supported package version here https://github.com/Microsoft/vstest/blob/master/src/Microsoft.TestPlatform.ObjectModel/Microsoft.TestPlatform.ObjectModel.csproj#L37
     -->
    <PackageReference Include="System.Reflection.Metadata" Version="1.5.0" />
    <PackageReference Include="Microsoft.Extensions.FileSystemGlobbing" Version="2.0.1" />
```

Reply:
Comments and reference moved to target file. ```Metadata```  and ```FileSystemGlobbing``` was just missed by me.